.github/workflows/release-nightly.yml

```diff
@@ -3,6 +3,9 @@ on:
   push:
     branches:
       - master
+  pull_request:
+    branches:
+      - master
```

Comment:
This is a serious error, a very PR to master now release the nightly bridge package. As you can see, this PR released a new bridge package 13 minutes ago https://github.com/axieinfinity/bridge-v2/pkgs/container/bridge. Please remove this as soon as possible.